src/libraries/System.ComponentModel.Annotations/src/System/ComponentModel/DataAnnotations/Validator.cs

```diff
@@ -137,7 +137,6 @@ public static class Validator
                 throw new ArgumentNullException(nameof(instance));
             }
 
-            // TODO-NULLABLE: null validationContext isn't supported (GetObjectValidationErrors will throw), remove that check
```

Comment:
removing the check will cause NRE in the same line if it's actually null - it's true there is second null check in mentioned method though